source/_integrations/plugwise.markdown

```diff
@@ -151,3 +151,9 @@ Smile P1 (DSMR):
  - v4.0
  - v3.3
  - v2.5
+ - v2.1
+
+Stretch (power switches):
+
+ - v3.1
+ - v2.3
```

Comment:
Maybe we should write here v1.x, 2.x, etc. as we now only check the major version-number in Plugwise-Smile.